Rakefile

```diff
@@ -155,3 +155,26 @@ namespace :release do
     end
   end
 end
+
+namespace :pkg do
+  desc 'Sync the root rust-toolchain version to all crates'
+  task :'rust_version:sync' do
```

Comment:
this is awesome! Can you update the list of rake tasks in `CONTRIBUTING.md`?

Please run `bundle exec rake --tasks` and paste the output here:

https://github.com/artichoke/artichoke/blob/53e96d0a593983316e464eb0e3713a542b06ab45/CONTRIBUTING.md#L61-L86